driver/mounter.go

```diff
@@ -102,8 +125,9 @@ func newMounter(log *logrus.Entry) *mounter {
 	}
 
 	return &mounter{
-		kMounter: kMounter,
-		log:      log,
+		kMounter:            kMounter,
+		log:                 log,
+		attachmentValidator: &(prodAttachmentValidator{}),
```

Comment:
The parentheses `()` should not be required.

Reply:
Removed.